### ISSUE TITLE
Invitation QR code too large on high res android tablet

### ISSUE THREAD

**Issue by chrisamow** [simplex-chat/simplex-chat#1439]
Create one-time invitation link on a high res tablet (higher res than 1080p anyway) like:
Lenovo Tab P11 Plus
Screen resolution: 2000x1200
Android 11

It scaled up so large only a part of it (approx a third?) is on the screen and so can't be scanned.  I couldn't find a way to pinch scale it down or make it size to screen.

![image](https://user-images.githubusercontent.com/5819969/204081737-d0632e09-3032-4bdb-bea8-f00049400c0e.png)

(Doing the invite from the other side worked fine, so happy ending - great work, this project is awesome!)

**Comment by epoberezkin** [comment]
Yes, the app isnt prepared for horizontal layout in general - I think it should fit in vertical though?

**Comment by chrisamow** [comment]
> 

Yes - vertical is looking good - nice!
Have a good one!!  
(I'll leave it up to you whether to keep this open, or let me know and I'll close it.)
